openfe/utils/handle_trajectories.py

```diff
@@ -0,0 +1,207 @@
+                              frame_num: int) -> unit.Quantity:
+    """
+    Helper method to extract atom positions of a state at a given frame.
+
+    Parameters
+    ----------
+    dataset : netCDF4.Dataset
+        Dataset containing the MultiState information.
+    state_num : int
+        State index to extract positions for.
+    frame_num : int
+        Frame number to extract positions for.
+
+    Returns
+    -------
+    unit.Quantity
+        n_atoms * 3 position Quantity array
+    """
+    effective_replica = _effective_replica(dataset, state_num, frame_num)
+    pos = dataset.variables['positions'][frame_num][effective_replica].data
+    pos_units = dataset.variables['positions'].units
+    return pos * unit(pos_units)
+
+
+def _create_new_dataset(filename: Path, n_atoms: int,
+                        title: str) -> nc.Dataset:
+    """
+    Helper method to create a new NetCDF dataset which follows the
+    AMBER convention (see: https://ambermd.org/netcdf/nctraj.xhtml)
+
+    Parameters
+    ----------
+    filename : path.Pathlib
+        Name of the new netcdf trajectory to write.
+    n_atoms : int
+        Number of atoms to store in trajectory.
+    title : str
+        Title of trajectory.
+
+    Returns
+    -------
+    netCDF4.Dataset
+        AMBER Conventions compliant NetCDF dataset to store information
+        contained in MultiState reporter generated NetCDF file.
+    """
+    ncfile = nc.Dataset(filename, 'w', format='NETCDF3_64BIT')
+    ncfile.Conventions = 'AMBER'
+    ncfile.ConventionVersion = "1.0"
+    ncfile.application = "openfe"
+    ncfile.program = f"openfe {__version__}"
+    ncfile.programVersion = f"{__version__}"
+    ncfile.title = title
+    
+    # Set the dimensions
+    ncfile.createDimension('frame', None)
+    ncfile.createDimension('spatial', 3)
+    ncfile.createDimension('atom', n_atoms)
+    ncfile.createDimension('cell_spatial', 3)
+    ncfile.createDimension('cell_angular', 3)
+    ncfile.createDimension('label', 5)
+    
+    # Set the variables
+    # positions
+    pos = ncfile.createVariable('coordinates', 'f4', ('frame', 'atom', 'spatial'))
+    pos.units = 'angstrom'
+    # we could also set this to 0.1 and do no nm to angstrom scaling on write
+    pos.scale_factor = 1.0 
+
+    # Note: OpenMMTools NetCDF files store velocities
+    # but honestly it's rather useless, so we don't populate them
+    # Note 2: NetCDF file doesn't contain any time information... 
+    # so we can't populate that either, this might trip up some readers..
+    # Note 3: We'll need to convert box vectors (in nm) to
+    # unitcell (in angstrom & degrees)
+    cell_lengths = ncfile.createVariable(
+        'cell_lengths', 'f8', ('frame', 'cell_spatial')
+    )
+    cell_lengths.units = 'angstrom'
+    cell_angles = ncfile.createVariable(
+        'cell_angles', 'f8', ('frame', 'cell_spatial')
+    )
+    cell_angles.units = 'degree'
+    
+    return ncfile
+
+
+def _get_unitcell(
```

Comment:
So the coordinates are coming out reasonable for the individual molecules, but it looks like there's some kind of imaging issue? I'm not sure that I didn't mess up the unitcell conversion tbh.

![image](https://github.com/OpenFreeEnergy/openfe/assets/12460125/d2cb125e-46ca-4d24-b64d-4b368b1004b0)